docs/using_unidist/unidist_on_dask.rst

```diff
@@ -0,0 +1,37 @@
+..
+      Copyright (C) 2021-2022 Modin authors
+
+      SPDX-License-Identifier: Apache-2.0
+
+:orphan:
+
+unidist on Dask
+'''''''''''''''
+
+This section describes the use of unidist with the Dask execution backend.
+
+There are two ways to choose the execution backend to run on.
+First, the recommended way is to use the argument of unidist CLI:
+
+.. code-block:: bash
+
+    # Running the script with unidist on Dask backend
+    $ unidist script.py --backend dask
+
+For more information on the CLI arguments specific to the Dask backend
+see :doc:`unidist CLI </using_cli>` section.
+
+Second, setting the environment variable:
+
+.. code-block:: bash
+
+    # unidist will use Dask backend to distribute computations
+    export UNIDIST_BACKEND=dask
+
+For more information on the environment variables and associated configs specific to the Dask backend
+see :doc:`config API </flow/unidist/config>` section.
```

Comment:
Is it possible to make a link on specific subtitle from this section? (I mean the link to page https://unidist--140.org.readthedocs.build/en/140/flow/unidist/config.html#unidist-configuration-settings-list) (applicable for all files)

Reply:
I think we can leave as is